### PR TITLE
Feature/b2 c 144 email sent in change email flow

### PR DESCRIPTION
**Pull request by joseluisgraa**

### CODE REVIEW

File: b2c-app/src/pages/EmailSent.js [hunk 1]
```diff
@@ -30,29 +31,41 @@ class EmailSent extends React.Component {
         this.setState({ showErrors: true });
     }
 
-    async resendEmail(e) {
+    clearPageLevelError() {
+        this.setState({ errors: [] });
```
Comment (DrogoNevets) [inline b2c-app/src/pages/EmailSent.js:35]: merge both lines into one as setState is async and will cause a double render

File: b2c-app/src/pages/EmailSent.js [hunk 2]
```diff
@@ -30,29 +31,41 @@ class EmailSent extends React.Component {
...
-            //stop spinner
-            this.setState({ showSpinner: false });
-        }
+        ChangeEmailService.callResendEmail().then(
+            () => {
+                //clear error messages if there were any from previous calls
+                this.clearPageLevelError();
+            },
+            (error) => {
+                if (error.userMessage) {
```
Comment (DrogoNevets) [inline b2c-app/src/pages/EmailSent.js:53]: do we need to hide spinner after an error?

Comment (joseluisgraa) [reply b2c-app/src/pages/EmailSent.js:53]: Yes, it is done in the finally block

File: b2c-app/src/pages/EmailSent.js (hunk 1 above)
Comment (joseluisgraa) [reply b2c-app/src/pages/EmailSent.js]: done

**Pull request merged**